lib/assisted_workflow/addons/pivotal.rb

```diff
@@ -46,14 +45,24 @@ def pending_stories(options = {})
       log "loading pending stories"
       states = ["unstarted"]
       states << "started" if options[:include_started]
-      @project.stories.all(:state => states, :owned_by => @fullname, :limit => 5)
+      @project.stories(:filter => "state:#{states.join(',')} owned_by:#{@client.me.id}",  :limit => 5)
```

Comment:
Line is too long. [102/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.